src/Dropdown.vue

```diff
@@ -73,34 +78,48 @@ export default {
       return this.$parent && (this.$parent.menu || this.$parent.submenu)
     },
     slots () {
-      return this._slotContents
+      return this.$slots.content
     }
   },
   methods: {
     blur () {
       this.unblur()
       this._hide = setTimeout(() => {
         this._hide = null
-        this.show = false
+        this.hideDropdownMenu();
       }, 100)
     },
     unblur () {
       if (this._hide) {
         clearTimeout(this._hide)
         this._hide = null
       }
-    }
+    },
+    hideDropdownMenu() {
+      this.show = false;
+      const $el = $(this.$refs.dropdown);
+      $el.findChildren('ul').each(ul => ul.classList.remove('show'));
+    },
+    showDropdownMenu() {
+      this.show = true;
+      const $el = $(this.$refs.dropdown);
+      $el.findChildren('ul').each(ul => ul.classList.add('show'));
```

Comment:
- `this.$refs.dropdown` returns a HTML Element which can be manipulated directly, is there a reason to have `const $el`?
- Also maybe use `.toggle(class, true|false)`?

e.g.
`this.$refs.dropdown.findChildren('ul').each(ul => ul.classList.toggle('show', false));`